### PR TITLE
1.35 update and season 37 mod changes

### PR DESCRIPTION
**Pull request by EmilAminy**

### CODE REVIEW

File: sweddit_combined/common/estate_privileges/000_burgher_loan_fix.txt
Comment (EmilAminy) [inline sweddit_combined/common/estate_privileges/000_burgher_loan_fix.txt:1]: Paradox added the same change, we no longer need this

File: sweddit_combined/common/static_modifiers/sweddit.txt [hunk 1]
```diff
@@ -6,7 +6,6 @@ horde_unity = {
 	nomadic_tribes_loyalty_modifier = 0.05
 }
 
-# Drilling armies now gives 5% army professionalism per year
-drilling_armies = {
-	yearly_army_professionalism = 0.05
-}
```
Comment (EmilAminy) [inline sweddit_combined/common/static_modifiers/sweddit.txt:12]: Paradox changed how slacken works, this change is no longer needed

File: sweddit_combined/common/event_modifiers/sweddit_event_modifiers.txt [hunk 2]
```diff
@@ -1,5 +0,0 @@
-
-estate_cossacks_prime_herding_rights_mod = {
-    local_tax_modifier = -0.5
-    local_manpower_modifier = 0.25
-}
```
Comment (EmilAminy) [inline sweddit_combined/common/event_modifiers/sweddit_event_modifiers.txt:5]: Paradox added the same change, we no longer need this

File: sweddit_combined/common/on_actions/sweddit_on_actions.txt [hunk 3]
```diff
@@ -1,4 +0,0 @@
-# Sweddit change: Gain 5 professionalism per general
-on_general_recruited = {
-	add_army_professionalism = 0.05
-}
```
Comment (EmilAminy) [inline sweddit_combined/common/on_actions/sweddit_on_actions.txt:4]: Paradox changed how slacken works, this change is no longer needed

File: sweddit_combined/common/defines/sweddit.lua [hunk 4]
```diff
@@ -2,7 +2,6 @@ NDefines.NCountry.COUNTRIES_GETTING_SCORE = 0
 NDefines.NMilitary.WARGOAL_MAX_BONUS = 35
 NDefines.NDiplomacy.MIN_RELATIONS_TO_ALLY = -200
 
-NDefines.NMilitary.SLACKEN_AP_DROP = 0.25
```
Comment (EmilAminy) [inline sweddit_combined/common/defines/sweddit.lua:5]: Paradox changed how slacken works, this change is no longer needed

File: README.md [hunk 5]
```diff
@@ -3,59 +3,44 @@
...
 
 #### Triggered Modifiers
-- Added modifier, all **Theocracies** with **Confucian** religion gain **+5% morale of armies** and **+10% goods produced**
+- The **Chinese emperor** always get **+0.1 mandate growth**
 
-#### Wargoals
-- Changed **Unify China** CB. Now only available **before 1550** to countries with **Chinese culture group primary culture** or **Manchu tag** or **Qing tag**. Applies to all provinces within **China super region**. Grants **75% aggressive expansion**, **150% prestige** and **100% peace cost**
+## Alliance mod
+
+- Custom system which keeps track of player-player alliances
```
Comment (EmilAminy) [inline README.md:17]: This change is not added in this MR, gurkan will add in separate MR. Preemptively  added it to changelog.

File: sweddit_combined/common/estate_privileges/000_burgher_loan_fix.txt [hunk 6]
```diff
@@ -1,101 +0,0 @@
...
-	}
-	penalties = {
-		trade_efficiency = -0.05
-	}
-	benefits = {
-	}
-	ai_will_do = {
-		factor = 0
-	}
-}
```
Comment (EmilAminy) [inline sweddit_combined/common/estate_privileges/000_burgher_loan_fix.txt:101]: Paradox added the same change, we no longer need this

File: sweddit_combined/events/as_events.txt [hunk 7]
```diff
@@ -0,0 +1,245 @@
...
+                }
+            }
+            add_country_modifier = {
+                name = as_notify_timer
+                duration = 90
+            }
+            every_country = {
+                limit = {
+                    NOT = {
+                        tag = ROOT
```
Comment (Sepax) [inline sweddit_combined/events/as_events.txt:182]: add is_player = yes

File: sweddit_combined/common/scripted_effects/as_scripted_effects.txt [hunk 8]
```diff
@@ -0,0 +1,541 @@
...
+		}
+		subtract_variable = {
+			which = as_points_max_puny_power
+			which = as_cost_puny_power
+		}
+	}
+}
+
+update_ratings = {
+	every_country = {
```
Comment (Sepax) [inline sweddit_combined/common/scripted_effects/as_scripted_effects.txt:123]: limit = {
    is_player = yes
    NOT = {
        tag = CZH
    }
}

**Review by ludlyl — APPROVED** [review]

**Pull request merged**